code/modules/mob/living/simple_animal/hostile/cockroach.dm

```diff
@@ -66,6 +66,11 @@
 		if(isliving(AM))
 			var/mob/living/A = AM
 			if(A.mob_size > MOB_SIZE_SMALL && !(A.movement_type & FLYING))
+				if(ishuman(A))
+					var/mob/living/carbon/human/H = A
+					if(HAS_TRAIT(H, TRAIT_PACIFISM))
+						H.visible_message("<span class='notice'>[src] avoids getting crushed.</span>", "<span class='warning'>You avoid crushing [src]!</span>")
```

Comment:
```suggestion
						H.visible_message("<span class='notice'>[H] carefully steps over [src].</span>", "<span class='notice'>You carefully step over [src] to avoid hurting it.</span>")
```

I added this in #47352. In hindsight I should've hinted it was caused by pacifism.